### DISCUSSION TITLE
In which branch should I fork for this project?

### DISCUSSION THREAD

**Discussion by otnansirk** [devvsakib/hacktoberfest-react-project#210] [Q&A] [answered]
In which branch should I fork for this project?

**Comment by devvsakib** [comment, marked as answer]
> In which branch should I fork for this project?

master branch.

**Reply by otnansirk** [reply]
master branch ? 
I don't see any branch with the name master here.  https://github.com/devvsakib/power-the-web/branches

do you mean `main` branch ?

**Reply by devvsakib** [reply]
oh yes. I just messed up with another repository. you can fork and clone it. No need to worry about the branch

**Reply by otnansirk** [reply]
oh right. thank you
